### PR TITLE
Fix coding standards

### PR DESCRIPTION
**Pull request by samwilson**
This brings things into alignment with MediaWiki coding standards
and also ignores local phpunit.xml files.

There are two remaining warnings, about using `@` silencing in MediawikiSession and I'm not sure how these should be resolved.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,9 +27,11 @@
 		"phpunit-unit": "phpunit --testsuite unit",
 		"phpunit-integration": "phpunit --testsuite integration",
 		"phpunit-coverage": "phpunit --coverage-clover=coverage.clover",
+		"phpcs": "phpcs -ps",
 		"test": [
 			"@lint",
-			"@phpunit"
+			"@phpunit",
```
Comment (addshore) [inline composer.json]: small note, could we put phpcs before phpunit? :)

Comment (samwilson) [reply composer.json]: Good point, then it will match the order that Travis is doing it too. And fail quicker.

**Comment by addshore** [comment]
@samwilson looks like we should do https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file for the things in MediawikiSession and then merge this!

**Comment by samwilson** [comment]
@addshore good point. I guess I wasn't quite sure if it'd be best to ignore the error-ignoring, or fix it with a try-catch block or something. But I've `@codingStandardsIgnoreLine`'d them for now.

**Pull request merged**